wondrous-app/pages/organization/[username]/members.tsx

```diff
@@ -23,11 +24,20 @@ const useGetOrgFromUsername = (username) => {
 
 function OrgMemberPage() {
   const router = useRouter();
+  const { setPageData } = usePageDataContext();
   const { username } = router.query;
   const { data: userPermissionsContext } = useQuery(GET_USER_PERMISSION_CONTEXT, {
     fetchPolicy: 'cache-and-network',
   });
   const org = useGetOrgFromUsername(username);
+  useEffect(() => {
+    if (org) {
+      setPageData({ orgData: org });
+    }
+  }, [org]);
+
+  useEffect(() => () => setPageData({}), [])
```

Comment:
is the point here to reset it every time?

Reply:
yes, on page change unmount we need to update the state for the dropdown to update accordingly

Reply:
ah